### DISCUSSION TITLE
How get input from terminal into and response send back to terminal

### DISCUSSION THREAD

**Discussion by muhammad-abrar-hussain** [tzfun/vue-web-terminal#120] [Q&A]
Hi, 
I am experiencing an issue where the command I type in the terminal interface is not being received in the body of the component that uses the vue-web-terminal component. How I can receive command into the body of the component and send back response from component to terminal? 
Your support will be highly appreciated.
Thanks

**Comment by tzfun** [comment]
I don't know if I understand what you mean, I guess you don't know how to use vue-web-terminal to implement input and output? If so, you need to focus on this very important event: [exec-cmd](https://tzfun.github.io/vue-web-terminal/events.html#exec-cmd)

You can refer to this [simple example](https://tzfun.github.io/vue-web-terminal/getting-started.html#your-first-vue-web-terminal).

For slightly more complex scenarios, you can refer to the code that implements the effects in the [demo](https://tzfun.github.io/vue-web-terminal/demo.html): [LocalTerminal.vue](https://github.com/tzfun/vue-web-terminal/blob/doc/docs/.vuepress/components/local-terminal/LocalTerminal.vue)
